jdaviz/configs/mosviz/helper.py

```diff
@@ -73,7 +139,7 @@ def add_column(self, data, column_name=None):
         column_name : str
             Header string to be shown in the table.
         """
-        table_data = self.app.data_collection['MOS Table']
+        table_data = self.app.data_collection["MOS Table"]
```

Comment:
```suggestion
        table_data = self.app.data_collection['MOS Table']
```
This has not been "set in stone", so to speak, but the unspoken formalism so far has been, essentially, double quotes for text, especially text that will be used with string interpolation, and single quotes for anything that behaves like an identifier (e.g. `MOS Table` in this case).

It has been more-or-less up to developers to decide how to use their single or double quotes, but I think the basic rule is that we shouldn't tread on each other's chosen formalism if it doesn't conflict with some dev-team codebase-level standards.

Reply:
Ahh oops, that's my bad, and wasn't a conscious decision. I have VScode configured to automatically format my code on save with the [black formatter](https://black.readthedocs.io/en/stable/), as that was the one we were experimenting with in Green Bank. Totally agree with your statement, I should switch that to something less aggressive